entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt

```diff
@@ -32,9 +34,9 @@ class EntityHandler(
     /**
      * Implements 6.4.3.1 - Create Entity
      */
-    fun create(req: ServerRequest): Mono<ServerResponse> {
-
-        return req.bodyToMono<String>()
+    @PostMapping(consumes = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])

```

Comment:
I'm wondering if the `consumes` can't declared at a global level (per HTTP method maybe)?

Reply:
I'll check if it's  possible to do it

Reply:
can't find a way to declare this globally

Reply:
ok, let's start with this version then.

Reply:
I may have an idea to achieve a "global declaration". We could define a new Annotation which is annotated by the PostMapping annotation itself : 
See a draft : 
```java
@PostMapping(consumes = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])
public @interface PostJsonLDMapping { 
    @AliasFor(annotation = PostMapping.class, attribute = "value")
    String[] value() default {};
}
```
I don't know if it's worth giving it a try, I let you decide @HoucemKacem

Reply:
I tried it but it doesn't work since we cannot inherit the PostMapping annotation.
"@RequestMapping annotations are supported on the type and method-level, where method-level annotations inherit or extend type-level metadata and having an additional dimension is a significantly more complex model." from an issue in spring-framework

Reply:
This could be achieved with RequestMapping then. 
Maybe it's best to stick with what's done currently.